bzl/tbb.BUILD

```diff
@@ -1,67 +1,57 @@
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+load("@bazel_skylib//rules:copy_file.bzl", "copy_file")
+
 package(default_visibility = ["//visibility:public"])
 
 exports_files(["LICENSE"])
 
+write_file(
+    name = "gen_version_file",
+    out = "version_string.ver",
+    content = [
+        "#define __TBB_VERSION_STRINGS(N) \"Empty\"",
+    ],
+)
+
+copy_file(
+    name = "copy_asm_file",
+    src = "src/tbb/intel64-masm/intel64_misc.asm",
+    out = "src/tbb/intel64-masm/intel64_misc.S",
+)
+
 cc_library(
     name = "tbb",
     srcs = glob([
-        "tbb/**/*.cpp",
-        "tbb/**/*.cc",
-        "tbbmalloc/**/*.cpp",
-        "tbbmalloc/**/*.cc",
-    ]),
-    hdrs = glob([
-        "include/serial/**",
-        "include/tbb/**/**",
+        "src/tbb/*.cpp",
+        "include/tbb/*.h",
+        "src/tbb/*.h",
+        "src/rml/client/rml_tbb.cpp",
+        "src/rml/include/*.h",
+        ":copy_asm_file",
     ]),
-    includes = ["include"],
+    hdrs = [
+        ":gen_version_file",
+    ],
+    defines = [
+        "TBB_USE_THREADING_TOOLS",
+        "__TBB_DYNAMIC_LOAD_ENABLED=0",
+        "__TBB_SOURCE_DIRECTLY_INCLUDED=1",
+        "__TBB_x86_64=1",
+    ] + select({
+        "@bazel_tools//src/conditions:windows": [
```

Comment:
This conditional is not compatible with our build system.

Use `@com_intel_plaidml//toolchain:windows_x86_64` instead